usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -107,22 +122,24 @@ def construct_idv_response(requested_award_dict):
     response.update(award)
 
     parent_award = fetch_parent_award_details(award["generated_unique_award_id"])
+    transaction = fetch_fpds_details_by_pk(award["_trx"], mapper)
+
     response["parent_award"] = parent_award
     response["parent_generated_unique_award_id"] = parent_award["generated_unique_award_id"] if parent_award else None
     response["executive_details"] = fetch_officers_by_legal_entity_id(award["_lei"])
-    response["latest_transaction_contract_data"] = fetch_fpds_details_by_pk(award["_trx"], mapper)
+    response["latest_transaction_contract_data"] = transaction
     response["funding_agency"] = fetch_agency_details(response["_funding_agency"])
     response["awarding_agency"] = fetch_agency_details(response["_awarding_agency"])
     response["idv_dates"] = OrderedDict(
```

Comment:
`idv_dates` -> `period_of_performance` to match other responses

Reply:
Also, for Contracts and IDV responses, please add `potential_end_date` to the `period_of_performance` object